test/unit/config_noocsp_critical.cnf

```diff
@@ -0,0 +1,21 @@
+[ ca ]
```

Comment:
I wonder if all these resources the unit tests use (like manifests and these cnf config files) should be in a separate directory like `test/unit/data` or something like that, so they don't get mixed with the unit tests so it looks cleaner?

Reply:
Good point. fixed it/.